rails-homework/app/views/patients/index.html.erb

```diff
@@ -30,13 +30,19 @@
       <td> <%= patient.last_name %> </td>
       <td> <%= patient.email %> </td>
       <td> <%= patient.age %> </td>
-      <td>
-      <% if can? :update, @patients %>
-          <%= link_to "Edit", edit_patient_path(patient.id), class: "btn btn-outline-primary" %>
-      <% end %>
-      <% if can? :destroy, @patients %>
-          <%= link_to "Delete", patient_path(patient.id), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-outline-danger" %>
-      <% end %>
+      <td class="actions">
+        <% if can? :update, @patients %>
+            <%= link_to "Edit", edit_patient_path(patient.id), class: "btn btn-outline-primary" %>
+        <% end %>
+        <% if can? :destroy, @patients %>
+            <%= link_to "Delete", patient_path(patient.id), method: :delete, data: { confirm: "Are you sure?" }, class: "btn btn-outline-danger" %>
+        <% end %>
+        <% if can? :manage, @diagnosis_items %>
```

Comment:
Can view?

Reply:
`<% if can? :manage, @diagnosis_items %>` wrong?
I think `<% if can? :read, @diagnosis_items %>` would be better

Reply:
Yes